Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/azure-policy-deploy.yaml

```diff
@@ -31,32 +31,31 @@ spec:
       - name: azure-policy
         image: {{ .Values.azurepolicy.image.name }}:{{ .Values.azurepolicy.image.tag }}
         resources:
-{{ toYaml .Values.azurepolicy.resources.deployment | indent 9 }}
+          requests:
+            cpu: 30m
+            memory: 50Mi
+          limits:
+            cpu: 100m
+            memory: 200Mi
         imagePullPolicy: Always
         env:
         - name: K8S_POLICY_PREFIX
           value: azurepolicy
         - name: RESOURCE_ID
           value: {{ .Values.azurepolicy.env.resourceid }}
+        - name: RESOURCE_TYPE
+          value: Microsoft.Kubernetes/connectedClusters
         - name: CLIENT_ID
           value: {{ .Values.azurepolicy.env.clientid }}
         - name: TENANT_ID
           value: {{ .Values.azurepolicy.env.tenantid }}
-        - name: RESOURCE_TYPE
-          value: Microsoft.Kubernetes/connectedClusters
         - name: DATAPLANE_ENDPOINT
           value: https://gov-prod-policy-data.trafficmanager.net
-        - name: REFRESH_INTERVAL
-          value: 5m   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h"
-        - name: AUDIT_INTERVAL
-          value: 5m   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h"
-        - name: TELEMETRY_HEARTBEAT_INTERVAL
-          value: 5m   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h"
         - name: CLIENT_SECRET
           valueFrom:
             secretKeyRef:
               name: azure-policy
               key: client-secret
         - name: CURRENT_IMAGE
           value: {{ .Values.azurepolicy.image.name }}:{{ .Values.azurepolicy.image.tag }}
-{{- end }} 
+{{- end }}
```

Comment:
We should keep the newline at the end of the file

Reply:
Chart installation did work without the new line. Is it a hard requirement? And why is it needed?